src/components/generic/movies/style.tsx

```diff
@@ -11,7 +11,7 @@ const Movies = styled.div`
   justify-content: space-evenly;
 `
 
-const img = styled.div`
+export const img = styled.div<{src:string}>`
```

Comment:
Using `img` here is semantically wrong since you're not rendering an `img` element.

Reply:
Right, Changed that.

Reply:
Done  :white_check_mark: